CHANGELOG.md

```diff
@@ -17,6 +17,9 @@ nav_order: 1
 - Add new user config generator
 - Use `TypeSet` for `ip_filter`, `ip_filter_string` fields
 
+## [4.13.4] - 2024-02-08
+
+- Fix `aiven_organization_user_group` resource - `description` field is required
```

Comment:
Could you please remove the header (line 20) add you change (line 22) to the existing list (right after line 18). We put release number and the date when we make a new release :)